### DISCUSSION TITLE
System module  storage

### DISCUSSION THREAD

**Discussion by lujiayou123** [pyswmm/pyswmm#343] [Q&A]
![image](https://user-images.githubusercontent.com/25896865/171571797-c5bdaa46-04c4-4ece-83ee-969ca7bd9ee7.png)
![image](https://user-images.githubusercontent.com/25896865/171571747-436130cb-45e5-45d7-938d-4c5b8ddaca19.png)

Excuse me, How to implement the function in the above image by pyswmm?

**Comment by floriandorre** [comment]
Hi, you will probably find it using the output parser : https://pyswmm.readthedocs.io/en/latest/reference/output.html#pyswmm.output.Output.system_result
